source/jormungandr/jormungandr/street_network/handimap.py

```diff
@@ -54,10 +54,7 @@
     'url': 'https://www.handimap.fr',
 }
 
-
-class Languages(Enum):
-    french = "fr-FR"
-    english = "en-EN"
+LANGUAGE_TRANSFORMATION_LIST = {"en-US": "en-EN", "en-GB": "en-EN", "fr-FR": "fr-FR"}
```

Comment:
After previous conversation, it looks like this is not a transformation map, just a valid list.

Reply:
:+1: